crates/tools/js/benchmark/README.md

```diff
@@ -0,0 +1,21 @@
+# JS Benchmark Runner
+
+## Run
+
+To run:
+
+```shell
+pnpm install
+pnpm run benchmark
+```
+
+The measurements will be printed stdout as machine-readable json and to stderr
+as human-readable output.
```

Comment:
Formatting

Reply:
Thanks fixed in [daa0615](https://github.com/NomicFoundation/hardhat/pull/4978/commits/daa061553a1a33cecfe7e55f4fe978fc1e8980fc)

Reply:
Just one line was fixed. The whole file was reformatted